tt_metal/hw/inc/blackhole/core_config.h

```diff
@@ -47,3 +47,7 @@ constexpr uint8_t MaxProcessorsPerCoreType = 5;
 constexpr uint8_t NumTensixDispatchClasses = 3;
 constexpr uint8_t NumEthDispatchClasses = 1;
 constexpr uint8_t NumDramDispatchClasses = 1;
+constexpr uint8_t noc_size_x = 17;
+constexpr uint8_t noc_size_y = 12;
+#define ALLOCATOR_ALIGNMENT 32
```

Comment:
allocator alignment is 64 for BH because we take max of DRAM alignment (64) and L1 alignment (16)